src/charm.py

```diff
@@ -1001,7 +1001,10 @@ def delete_tls_certificate_from_workload(self) -> None:
             Config.TLS.INT_CA_FILE,
             Config.TLS.INT_PEM_FILE,
         ]:
-            container.remove_path(f"{Config.CONF_DIR}/{file}")
+            try:
+                container.remove_path(f"{Config.CONF_DIR}/{file}")
+            except PathError as err:
+                logger.debug("Path unavailable: %s (%s)", file, str(err))
```

Comment:
@delgod This was necessary to add (to have tests passing).

I.e. previously we only deleted certs is we surely had them available, now we may try to delete before any certs may be available. Is this OK?

Reply:
LGTM!